ProjectSettings/ProjectSettings.asset

```diff
@@ -796,7 +796,7 @@ PlayerSettings:
     PS4: UNITY_POST_PROCESSING_STACK_V2
     PS5: UNITY_POST_PROCESSING_STACK_V2
     Stadia: UNITY_POST_PROCESSING_STACK_V2
-    Standalone: UNITY_POST_PROCESSING_STACK_V2
+    Standalone: ENABLE_INTEGRATION_DISCORD
```

Comment:
I don't know what it does but is it ok if there is no longer UNITY_POST_PROCESSING_STACK_V2?

Reply:
UNITY_POST_PROCESSING_STACK_V2 is for the biRP, this is using URP which has post processing built in, meaning (unless it has a safety check) using anything from UNITY_POST_PROCESSING_STACK_V2 will make the world purple because of error shaders